lib/src/shared/main/java/com/couchbase/lite/internal/fleece/FLDict.java

```diff
@@ -56,25 +56,22 @@ public FLValue get(String key) {
     public Map<String, Object> asDict() {
         final Map<String, Object> results = new HashMap<>();
         final FLDictIterator itr = new FLDictIterator();
-        try {
-            itr.begin(this);
-            String key;
-            while ((key = itr.getKeyString()) != null) {
-                results.put(key, itr.getValue().asObject());
-                itr.next();
-            }
-        }
-        finally {
-            itr.free();
+
+        itr.begin(this);
+        String key;
+        while ((key = itr.getKeyString()) != null) {
+            results.put(key, itr.getValue().asObject());
+            itr.next();
         }
+
```

Comment:
Don't you need to free the FLDictIterator after usage?

Reply:
Well... its finalizer should free it, when it is GCed...